From c6e5a8bac0b32fb4f9131ea51f72372405a6e0b0 Mon Sep 17 00:00:00 2001
From: Antonio Alvarez Feijoo <antonio.feijoo@suse.com>
Date: Thu, 9 Feb 2023 13:54:46 +0100
Subject: [PATCH] fix(systemd): do not include systemd-random-seed.service

The systemd-random-seed.service unit is configured not to run in the initrd
since https://github.com/systemd/systemd/commit/fe7f113c

Also, the systemd-random-seed-load.service does not exist since
https://github.com/systemd/systemd/commit/c35b956d
---
 modules.d/00systemd/module-setup.sh | 2 --
 1 file changed, 2 deletions(-)

diff --git a/modules.d/00systemd/module-setup.sh b/modules.d/00systemd/module-setup.sh
index 2648ec4f..3d839217 100755
--- a/modules.d/00systemd/module-setup.sh
+++ b/modules.d/00systemd/module-setup.sh
@@ -106,8 +106,6 @@ install() {
         "$systemdsystemunitdir"/systemd-journald.service \
         "$systemdsystemunitdir"/systemd-vconsole-setup.service \
         "$systemdsystemunitdir"/systemd-volatile-root.service \
-        "$systemdsystemunitdir"/systemd-random-seed-load.service \
-        "$systemdsystemunitdir"/systemd-random-seed.service \
         "$systemdsystemunitdir"/systemd-sysctl.service \
         \
         "$systemdsystemunitdir"/sysinit.target.wants/systemd-modules-load.service \
-- 
2.43.0

